src/frontend/src/apis/tasks.api.ts

```diff
@@ -3,9 +3,16 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { WbsNumber, wbsPipe } from 'shared';
 import axios from '../utils/axios';
 import { apiUrls } from '../utils/urls';
 
+export const createSingleTask = (wbsNum: WbsNumber, payload: any) => {
```

Comment:
we want to move away from these any payload blocks and pass in the actual parameters